camera2gripper:
- - 0.0021889887182635626
  - -0.040122436652322936
  - 0.9991923730721085
  - -0.062088703965936604
- - -0.9999927986190336
  - 0.0030098512045020342
  - 0.0023116024311773333
  - 0.03175204824292044
- - -0.0031001674897306627
  - -0.9991902375788142
  - -0.04011555918504106
  - 0.11168222605352912
- - 0.0
  - 0.0
  - 0.0
  - 1.0
